src/main/java/Model/LadderSetting.java

```diff
@@ -0,0 +1,15 @@
+package Model;
+
+public class LadderSetting
+{
+    private final int width;
+    private final int height;
+
+    public LadderSetting(int width, int height) {
+        this.width = width-1;
+        this.height = height;
+    }
+
+    public int getWidth() { return width; }
+    public int getHeight() { return height; }
```

Comment:
```suggestion
    public int getWidth() { 
        return width; 
    }
    
    public int getHeight() { 
        return height; 
    }
```

이런식으로 해주면 가독성이 좋을 것 같아요!

Reply:
아 이거 IntelliJ에서 저렇게 축약해서 보여주길래 저렇게 해두는게 좋은건가 싶어서 해뒀는데, 역시 늘여쓰는게 맞나보네요 ㅎ...